src/OpenCL/Extensions/Silk.NET.OpenCL.Extensions.INTEL/IntelSharingFormatQueryGl.gen.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using System.Text;
+using Silk.NET.Core;
+using Silk.NET.Core.Native;
+using Silk.NET.Core.Attributes;
+using Silk.NET.Core.Contexts;
+using Silk.NET.Core.Loader;
+using Silk.NET.OpenCL;
+using Extension = Silk.NET.Core.Attributes.ExtensionAttribute;
+
+#pragma warning disable 1591
+
+namespace Silk.NET.OpenCL.Extensions.INTEL
+{
+    [Extension("INTEL_sharing_format_query_gl")]
+    public unsafe partial class IntelSharingFormatQueryGl : NativeExtension<CL>
+    {
+        public const string ExtensionName = "INTEL_sharing_format_query_gl";
+        [NativeApi(EntryPoint = "clGetSupportedGLTextureFormatsINTEL")]
+        public unsafe partial int GetSupportedGltextureFormats([Flow(FlowDirection.In)] nint context, [Flow(FlowDirection.In)] INTEL flags, [Flow(FlowDirection.In)] uint image_type, [Flow(FlowDirection.In)] uint num_entries, [Flow(FlowDirection.Out)] uint* gl_formats, [Flow(FlowDirection.Out)] uint* num_texture_formats);
+
+        [NativeApi(EntryPoint = "clGetSupportedGLTextureFormatsINTEL")]
+        public unsafe partial int GetSupportedGltextureFormats([Flow(FlowDirection.In)] nint context, [Flow(FlowDirection.In)] INTEL flags, [Flow(FlowDirection.In)] uint image_type, [Flow(FlowDirection.In)] uint num_entries, [Flow(FlowDirection.Out)] uint* gl_formats, [Flow(FlowDirection.Out)] out uint num_texture_formats);
+
+        [NativeApi(EntryPoint = "clGetSupportedGLTextureFormatsINTEL")]
+        public unsafe partial int GetSupportedGltextureFormats([Flow(FlowDirection.In)] nint context, [Flow(FlowDirection.In)] INTEL flags, [Flow(FlowDirection.In)] uint image_type, [Flow(FlowDirection.In)] uint num_entries, [Flow(FlowDirection.Out)] out uint gl_formats, [Flow(FlowDirection.Out)] uint* num_texture_formats);
+
+        [NativeApi(EntryPoint = "clGetSupportedGLTextureFormatsINTEL")]
+        public partial int GetSupportedGltextureFormats([Flow(FlowDirection.In)] nint context, [Flow(FlowDirection.In)] INTEL flags, [Flow(FlowDirection.In)] uint image_type, [Flow(FlowDirection.In)] uint num_entries, [Flow(FlowDirection.Out)] out uint gl_formats, [Flow(FlowDirection.Out)] out uint num_texture_formats);
```

Comment:
gl_formats -> glFormats? Is this a new issue or have I just never noticed (changing parameter names is a breaking change)

Reply:
This is exactly how it's defined upstream. We don't do any name modifications to parameter names.